<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="4" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-11T03:40:11 IST" name="Suite" finished-at="2023-02-11T03:40:30 IST" duration-ms="18451">
    <groups>
    </groups>
    <test started-at="2023-02-11T03:40:11 IST" name="Test" finished-at="2023-02-11T03:40:30 IST" duration-ms="18451">
      <class name="testcases.AdminMenuTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.AdminMenuTest@121314f7]" started-at="2023-02-11T03:40:18 IST" name="setup" finished-at="2023-02-11T03:40:22 IST" duration-ms="4466" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyAddUser(java.util.Map)[pri:0, instance:testcases.AdminMenuTest@121314f7]" started-at="2023-02-11T03:40:22 IST" name="verifyAddUser" data-provider="addUser" finished-at="2023-02-11T03:40:30 IST" duration-ms="7210" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Secret Question=City Of Birth, Email=testauto1@gmail.com, User Name=TestAuto1, Confirm Password=Test@123, City=Ghaziabad, Mobile=8899776655, Secret Question Answer=Delhi, State=UTTAR PRADESH, Address 1=A-74, PIN Code=201206, District=GHAZIABAD, Password=Test@123}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAddUser -->
      </class> <!-- testcases.AdminMenuTest -->
      <class name="testcases.LoginPageTest">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:testcases.LoginPageTest@242b836]" started-at="2023-02-11T03:40:11 IST" name="beforeSuite" finished-at="2023-02-11T03:40:11 IST" duration-ms="371" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.LoginPageTest@242b836]" started-at="2023-02-11T03:40:11 IST" name="setup" finished-at="2023-02-11T03:40:15 IST" duration-ms="3384" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginPage()[pri:0, instance:testcases.LoginPageTest@242b836]" started-at="2023-02-11T03:40:15 IST" name="verifyLoginPage" finished-at="2023-02-11T03:40:16 IST" duration-ms="1196" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at testcases.LoginPageTest.verifyLoginPage(LoginPageTest.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPage -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:testcases.LoginPageTest@242b836]" started-at="2023-02-11T03:40:30 IST" name="afterSuite" finished-at="2023-02-11T03:40:31 IST" duration-ms="1340" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testcases.LoginPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
